### PR TITLE
Let's try this again

### PR DESCRIPTION
**Pull request by lucywyman**
I'm not sure whether this aligns with your vision for the project, but I added everything to it's own file, so you can integrate as desired. Let me know if you want more help with it!

### CODE REVIEW

**Comment by pop** [comment]
Could you describe the changes you made? I'm not sure what I'm looking at. 

(also its README.rst not README.md, just a minor tweak)

**Comment by lucywyman** [comment]
- Included manifest file to specify default files in distributed package
- Added copy_file and copy_dir to, well, copy files and directories from a source to destination
- Modified grab to download files to a data directory created in the users current directory (TODO allow them to specify a directory)
- Modified build to write output to it's own directory
- From os import ... -> import os (I think I tried to use path as a variable, then ended up not doing that because it was ambiguous, though I think this is still good practice)
- Added main function for testing

**Comment by pop** [comment]
@lucywyman I appreciate the PR, unfortunately I don't really think it aligns with the design principles of `pageup`.
- The build and data dirs are a good idea (I mean... they're used in pretty much every popular SSG I can think of) but pageup is meant to be a lot more simple. If you're making a project that requires a `build` dir, pageup is probably not what you're looking for. It also muddies up the URL schema I have (with a build dir either the user has to move the files after building or the url looks like `example.com/mypage/build/` instead of just `example.com/mypage/`)
  - If we're not creating a data and build dir the use of a copy_file and copy_dir directory isn't necessary, although they would be good methods to have.
- The use of urllib is fine as it is cleaner code than using requests, but we actually migrated away from requests early on as it was suggested by some peers that it's not a good library for downloading files. Granted, the user needs to download requests to make pageup go, but they also need to download jinja2 and markupsafe so might as well throw in another useful lib while we're at it. (If you're gonna do it, do it right).
- As far as `import os` instead of `from os import path` I actually try to only include what I need form libraries, this is mostly a design choice.
- Also the use of a `pageup` script instead of piling everything into `pageup.py` is so the `pageup.py` acts as a `lib.py` file (I should really just name it that) and `pageup` is a script that calls what the user wants (`build` vs `init`).
- I also think this PR eliminates the ability to specify custom `init directory` choices and only allows for using the current working directory as the site dir. This feature is important to me so I don't want to get rid of it.

TLDR: Thank you for the Pull Request, but it I'm going to pass. I appreciate your time :smile:
